src/app/time-entry/time-entry.component.scss

```diff
@@ -9,11 +7,10 @@ form{
     border: 0.25px white solid;
   }
 
-  // button{
-  //   color: #fff;
-  //   background-color: #03a9f4;
-  //   border-color: #03a9f4;
-  // }
+}
+
+input{
```

Comment:
https://github.com/rohan-compro/clockify-clone/pull/2/files#r923011690

and remove extra lines